assets/locales/ru/translation.json

```diff
@@ -0,0 +1,159 @@
+    "Minimize": "Свернуть",
+    "Close": "Закрыть",
+    "Bring All to Front": "Показать все",
+    "Help": "Помощь",
+    "Documentation": "Документация",
+    "Search Issues": "Поиск дефектов",
+    "About Appium": "Об Appium",
+    "Check for updates": "Проверить обновления",
+    "New Session Window…": "Окно новой сессии…",
+    "Configurations…": "Настройки…",
+    "Hide Appium": "Скрыть Appium",
+    "Hide Others": "Скрыть другие",
+    "Show All": "Показать все",
+    "Quit": "Выход",
+    "Toggle Full Screen": "Включить полноэкранный режим",
+    "Languages": "Языки",
+    "View": "Экран",
+    "Reload": "Перезагрузить",
+    "Toggle Developer Tools": "Инструменты разработчика",
+    "Learn More": "Узнать больше",
+    "Undo": "Отменить",
+    "Cloud Providers": "Облачные провайдеры",
+    "Environment Variables": "Переменные окружения",
+    "Save and Restart": "Сохранить и перезапустить",
+    "Application must be restarted for changes to take effect": "Перезапустите приложение, чтобы изменения вступили в силу",
+    "Restart Now": "Перезапустить сейчас",
+    "Restart Later": "Перезапустить позже",
+    "Simple": "Обычный",
+    "Advanced": "Продвинутый",
+    "Presets": "Предустановки",
+    "Edit Configurations": "Изменить настройки",
+    "Server Address": "Адрес сервера",
+    "Server Port": "Порт сервера",
+    "Logfile Path": "Путь к лог-файлу",
+    "Log Level": "Уровень логирования",
+    "Override Temp Path": "Переопределить путь к Temp",
+    "Node Config File Path": "Путь к файлу конфигуции Node",
+    "Local Timezone": "Местный часовой пояс",
+    "Allow Session Override": "Разрешить переопределение сессии",
+    "Log Timestamps": "Логирование отметок времени",
+    "Supress Log Color": "Убрать окрашивание логов",
+    "Strict Caps Mode": "Режим строгих возможностей",
+    "Relaxed Security": "Послабление безопасности",
+    "Default Capabilities": "Возможности по умолчанию",
+    "WebDriverAgent Port": "Порт WebDriverAgent",
+    "executeAsync Callback Host": "executeAsync Callback Host",
+    "executeAsync Callback Port": "executeAsync Callback Port",
+    "Bootstrap Port": "Порт Bootstrap",
+    "Selendroid Port": "Порт Selendroid",
+    "Chromedriver Port": "Порт Chromedriver",
+    "Chromedriver Binary Path": "Путь к файлу Chromedriver",
+    "Save Server Arguments Preset": "Сохранить конфигурацию параметров сервера",
+    "Save": "Сохранить",
+    "Cancel": "Отменить",
+    "Save As Preset…": "Сохранить как…",
+    "noPresets": "У Вас нет сохраненных конфигураций. Создайте новую на владке Продвинутый.",
+    "startServer": "Запустить сервер v{{serverVersion}}",
+    "deleteSelectedPresetConfirmation": "Вы уверены, что желаете удалить {{presetName}}?",
+    "savedTimestamp": "Сохранено {{when}}",
+    "General": "Общие",
+    "invalidType": "Неверный тип {{type}}",
+    "presetAdded": "Ваша конфигурация '{{presetName}}' была добавлена в список на вкладке Предустановки!",
+    "datetimeFormat": "MMM Do YYYY, h:mma",
```

Comment:
should the date format be also localized?

Reply:
I agree with that. Updated